src/Service/AlgoliaPageCrawler.php

```diff
@@ -64,6 +65,10 @@ public function getMainContent(): string
             $useXpath = false;
             $selector = $this->config()->get('content_element_tag');
         }
+        
+        $originalStage = Versioned::get_stage();
+        //Always set to live to ensure we don't pick up draft content in our render eg. draft elemental blocks
+        Versioned::set_stage('Live');
```

Comment:
@josephlewisnz in these cases could you please use `Versioned::set_stage(Versioned::LIVE)` just prevents any issues.

Reply:
I've added that  now - I'll do some additional testing today